microprofile/server/src/main/java/io/helidon/microprofile/server/Server.java

```diff
@@ -148,6 +148,8 @@ static Builder builder() {
      * Builder to build {@link Server} instance.
      */
     final class Builder {
+        private static final long STARTUP_TIME = ServerImpl.STARTUP_TIME;
+
```

Comment:
Where is this constant used?

Reply:
The goal is to load the `ServerImpl` class as early as possible so the `STARTUP_TIME` constant there is set as early as possible. We could replace this constant in the `Builder` with a static initializer in the builder to accomplish the same thing.